client/components/notice/test/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+import React from 'react';
+import { shallow } from 'enzyme';
+import { identity } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { Notice } from '../index';
+
+describe( 'Notice', function() {
+	it( 'should output the component', function() {
+		const wrapper = shallow( <Notice translate={ identity } /> );
+		assert.isOk( true, wrapper.find( '.notice' ).length );
```

Comment:
With the first argument still present, the test is ineffective. You'll see even if the length was 0, it would still pass:

```diff
diff --git a/client/components/notice/test/index.js b/client/components/notice/test/index.js
index 7a24caa..ae349dc 100644
--- a/client/components/notice/test/index.js
+++ b/client/components/notice/test/index.js
@@ -14,7 +14,7 @@ import { Notice } from '../index';
 describe( 'Notice', function() {
        it( 'should output the component', function() {
                const wrapper = shallow( <Notice translate={ identity } /> );
-               assert.isOk( true, wrapper.find( '.notice' ).length );
+               assert.isOk( true, 0 );
        } );
 
        it( 'should have dismiss button when showDismiss passed as true', function() {
```

Dropping the `true` argument from each test should fix it up.